src/components/reporter/Form.js

```diff
@@ -318,8 +319,31 @@ const Form = ({ contest, sponsor, repoUrl }) => {
         });
 
         return (
-          <div className={clsx(styles.Form)}>
-            <h1>{sponsor} contest finding</h1>
+          <div
+            className={
+              !isExpanded
+                ? clsx(styles.Form)
+                : clsx(styles.Form, styles.FormMax)
+            }
+          >
+            <div className={clsx(styles.FormHeader)}>
+              <h1>{sponsor} contest finding</h1>
+                {isExpanded ? (
+                  <img
+                    src="/images/compress.svg"
+                    alt="compress form"
+                    className={clsx(styles.FormIcons)}
+                    onClick={() => setIsExpanded(false)}
+                  />
+                ) : (
+                  <img
+                    src="/images/expand.svg"
+                    alt="expand form"
+                    className={clsx(styles.FormIcons)}
+                    onClick={() => setIsExpanded(true)}
+                  />
+                )}
```

Comment:
While we're waiting on icon approval.. what do you think of this form?

```suggestion
                <img
                  src={isExpanded ? "/images/compress.svg" : "/images/expand.svg"}
                  alt={isExpanded ? "compress form" : "expand form"}
                  className={clsx(styles.FormIcons)}
                  onClick={() => setIsExpanded(!isExpanded)}
                />
```

Reply:
Much better indeed :)